internal/server/recon/golden/recognize_places/result.json

```diff
@@ -69,6 +69,38 @@
         }
       ]
     },
+    "I went to Half Moon Bay California and California, Washington County": {
+      "items": [
+        {
+          "span": "I went to"
+        },
+        {
+          "span": "Half Moon Bay California",
+          "places": [
+            {
+              "dcid": "geoId/06"
+            },
+            {
+              "dcid": "geoId/0631708"
+            }
+          ]
+        },
+        {
+          "span": "and"
+        },
+        {
+          "span": "California , Washington County",
+          "places": [
+            {
+              "dcid": "geoId/42125"
+            },
```

Comment:
ditto, why does it match county?

If this is a combined span, it should be only geoId/4210768, right?

Reply:
Done